util.sh

```diff
@@ -13,6 +13,8 @@ export -f vinfo
 
 
 function cleanup {
```

Comment:
Is this even used now? The Python script runs an equivalent?

Reply:
It is not.